src/fheroes2/heroes/heroes_dialog.cpp

```diff
@@ -344,9 +344,10 @@ int Heroes::OpenDialog( const bool readonly, const bool fade, const bool disable
 
     // Hero dismiss button.
     dst_pt.x = dialogRoi.x + 9;
-    dst_pt.y = dialogRoi.y + 318;
-    fheroes2::ButtonSprite buttonDismiss( dst_pt.x, dst_pt.y, fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ),
-                                          fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 1 ), fheroes2::AGG::GetICN( ICN::DISMISS_HERO_DISABLED_BUTTON, 0 ) );
+    dst_pt.y = dialogRoi.y + 378;
+    fheroes2::ButtonSprite buttonDismiss( dst_pt.x, dst_pt.y - fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ).height() / 2,
+                                          fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ), fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 1 ),
```

Comment:
We can call `GetICN()` once for `ICN::BUTTON_VERTICAL_DISMISS, 0`:
```suggestion
    const fheroes2::Sprite & dismissPressed = fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 );
    fheroes2::ButtonSprite buttonDismiss( dst_pt.x, dst_pt.y - dismissPressed.height() / 2,
                                          dismissPressed, fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 1 ),
```